backend/app/presenters/dividend_computation_presenter.rb

```
@@ -0,0 +1,37 @@
+    @dividend_computation = dividend_computation
+  end
+
+  def props
+    {
+      id: @dividend_computation.id,
+      total_amount_in_usd: @dividend_computation.total_amount_in_usd.to_f,
+      dividends_issuance_date: @dividend_computation.dividends_issuance_date.to_s,
+      return_of_capital: @dividend_computation.return_of_capital,
+      created_at: @dividend_computation.created_at.to_s,
+      confirmed_at: @dividend_computation.confirmed_at&.to_s,
+      outputs: @dividend_computation.dividend_computation_outputs.map { |output| output_props(output) }
```

RuboCop (1.75.5)

[convention] Put a comma after the last item of a multiline hash.

(Style/TrailingCommaInHashLiteral)